src/main/java/uk/gov/ons/fwmt/job_service/service/impl/JobProcessor.java

```diff
@@ -64,36 +64,80 @@ public void processSampleFile(File file) throws IOException {
     Iterator<CSVParseResult<LegacySampleIngest>> csvRowIterator = CSVParserBuilder.buildLegacySampleParserIterator(reader, filename.getTla(), fieldPeriodResourceServiceClient);
 
     while (csvRowIterator.hasNext()) {
+      
       CSVParseResult<LegacySampleIngest> row = csvRowIterator.next();
-      if (row.isError()) {
-        log.error("Job Entry could not be processed", FWMTCommonException.makeCsvOtherException(row.getErrorMessage()));
-        continue;
-      }
-
       final LegacySampleIngest ingest = row.getResult();
-      boolean isReallocation = jobResourceServiceClient.existsByTmJobId(ingest.getTmJobId());
-      //Don't change the jobtype string, this string is used in splunk report. if changing change splunk search query as well.
-      String jobType = isReallocation ? "Reallocation" : "Allocation";
-
+      boolean isExistingJob = jobResourceServiceClient.existsByTmJobId(ingest.getTmJobId());
```

Comment:
If row has error , we don't have to make the service call.
This call needs to be moved down or moved inside method